src/components/common/VHeaderControls.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="v-header-controls">
+    <v-client-id />
+    <v-logout-button class="v-header-controls-logout" />
+  </div>
+</template>
+
+<script>
+import VClientId from '@/components/common/VClientId';
+import VLogoutButton from '@/components/common/VLogoutButton';
+
+export default {
+  name: 'VHeaderControls',
+
+  components: {
+    VLogoutButton,
+    VClientId,
+  },
+};
+</script>
+
+<style lang="scss">
```

Comment:
`scoped`

Reply:
every component in this repo haven't scoped, just scss